diff -Naur a/Makefile.ffmpeg b/Makefile.ffmpeg
--- a/Makefile.ffmpeg	2022-01-12 15:10:35.168710941 +0200
+++ b/Makefile.ffmpeg	2022-01-14 17:36:35.167931179 +0200
@@ -183,7 +183,7 @@
 #
 
 $(LIB_ROOT)/$(NASM)/.tvh_download:
-	$(call DOWNLOAD,$(NASM_URL),$(LIB_ROOT)/$(NASM_TB),$(NASM_SHA1))
+#	$(call DOWNLOAD,$(NASM_URL),$(LIB_ROOT)/$(NASM_TB),$(NASM_SHA1))
 	$(call UNTAR,$(NASM_TB),z)
 	$(call PATCH,$(NASM),$(NASM_DIFFS))
 	@touch $@
@@ -223,7 +223,7 @@
 endif
 
 $(LIB_ROOT)/$(LIBX264)/.tvh_download:
-	$(call DOWNLOAD,$(LIBX264_URL),$(LIB_ROOT)/$(LIBX264_TB),$(LIBX264_SHA1))
+#	$(call DOWNLOAD,$(LIBX264_URL),$(LIB_ROOT)/$(LIBX264_TB),$(LIBX264_SHA1))
 	$(call UNTAR,$(LIBX264_TB),j)
 	@touch $@
 
@@ -273,7 +273,7 @@
 ifeq (yes,$(CONFIG_LIBX265_STATIC))
 
 $(LIB_ROOT)/$(LIBX265)/.tvh_download:
-	$(call DOWNLOAD,$(LIBX265_URL),$(LIB_ROOT)/$(LIBX265_TB),$(LIBX265_SHA1))
+#	$(call DOWNLOAD,$(LIBX265_URL),$(LIB_ROOT)/$(LIBX265_TB),$(LIBX265_SHA1))
 	$(call UNTAR,$(LIBX265_TB),z)
 	$(call PATCH,$(LIBX265),$(LIBX265_DIFFS))
 	@touch $@
@@ -325,7 +325,7 @@
 endif
 
 $(LIB_ROOT)/$(LIBVPX)/.tvh_download:
-	$(call DOWNLOAD,$(LIBVPX_URL),$(LIB_ROOT)/$(LIBVPX_TB),$(LIBVPX_SHA1))
+#	$(call DOWNLOAD,$(LIBVPX_URL),$(LIB_ROOT)/$(LIBVPX_TB),$(LIBVPX_SHA1))
 	$(call UNTAR,$(LIBVPX_TB),z)
 	@touch $@
 
@@ -363,7 +363,7 @@
 ifeq (yes,$(CONFIG_LIBOGG_STATIC))
 
 $(LIB_ROOT)/$(LIBOGG)/.tvh_download:
-	$(call DOWNLOAD,$(LIBOGG_URL),$(LIB_ROOT)/$(LIBOGG_TB),$(LIBOGG_SHA1))
+#	$(call DOWNLOAD,$(LIBOGG_URL),$(LIB_ROOT)/$(LIBOGG_TB),$(LIBOGG_SHA1))
 	$(call UNTAR,$(LIBOGG_TB),z)
 	@touch $@
 
@@ -401,7 +401,7 @@
 endif
 
 $(LIB_ROOT)/$(LIBTHEORA)/.tvh_download:
-	$(call DOWNLOAD,$(LIBTHEORA_URL),$(LIB_ROOT)/$(LIBTHEORA_TB),$(LIBTHEORA_SHA1))
+#	$(call DOWNLOAD,$(LIBTHEORA_URL),$(LIB_ROOT)/$(LIBTHEORA_TB),$(LIBTHEORA_SHA1))
 	$(call UNTAR,$(LIBTHEORA_TB),z)
 	@touch $@
 
@@ -449,7 +449,7 @@
 ifeq (yes,$(CONFIG_LIBVORBIS_STATIC))
 
 $(LIB_ROOT)/$(LIBVORBIS)/.tvh_download:
-	$(call DOWNLOAD,$(LIBVORBIS_URL),$(LIB_ROOT)/$(LIBVORBIS_TB),$(LIBVORBIS_SHA1))
+#	$(call DOWNLOAD,$(LIBVORBIS_URL),$(LIB_ROOT)/$(LIBVORBIS_TB),$(LIBVORBIS_SHA1))
 	$(call UNTAR,$(LIBVORBIS_TB),z)
 	@touch $@
 
@@ -493,7 +493,7 @@
 ifeq (yes,$(CONFIG_LIBFDKAAC_STATIC))
 
 $(LIB_ROOT)/$(LIBFDKAAC)/.tvh_download:
-	$(call DOWNLOAD,$(LIBFDKAAC_URL),$(LIB_ROOT)/$(LIBFDKAAC_TB),$(LIBFDKAAC_SHA1))
+#	$(call DOWNLOAD,$(LIBFDKAAC_URL),$(LIB_ROOT)/$(LIBFDKAAC_TB),$(LIBFDKAAC_SHA1))
 	$(call UNTAR,$(LIBFDKAAC_TB),z)
 	@touch $@
 
@@ -533,7 +533,7 @@
 ifeq (yes,$(CONFIG_LIBOPUS_STATIC))
 
 $(LIB_ROOT)/$(LIBOPUS)/.tvh_download:
-	$(call DOWNLOAD,$(LIBOPUS_URL),$(LIB_ROOT)/$(LIBOPUS_TB),$(LIBOPUS_SHA1))
+#	$(call DOWNLOAD,$(LIBOPUS_URL),$(LIB_ROOT)/$(LIBOPUS_TB),$(LIBOPUS_SHA1))
 	$(call UNTAR,$(LIBOPUS_TB),z)
 	@touch $@
 
@@ -577,7 +577,7 @@
 CONFIGURE_FFMPEG += --extra-ldflags=-L/usr/local/cuda/lib64
 
 $(LIB_ROOT)/$(FFNVCODEC)/.tvh_download:
-	$(call DOWNLOAD,$(FFNVCODEC_URL),$(LIB_ROOT)/$(FFNVCODEC_TB),$(FFNVCODEC_SHA1))
+#	$(call DOWNLOAD,$(FFNVCODEC_URL),$(LIB_ROOT)/$(FFNVCODEC_TB),$(FFNVCODEC_SHA1))
 	$(call UNTAR,$(FFNVCODEC_TB),z)
 	@touch $@
 
@@ -674,7 +674,7 @@
 
 $(LIB_ROOT)/$(FFMPEG)/.tvh_download:
 	@mkdir -p $(EBUILDIR)
-	$(call DOWNLOAD,$(FFMPEG_URL),$(LIB_ROOT)/$(FFMPEG_TB),$(FFMPEG_SHA1))
+#	$(call DOWNLOAD,$(FFMPEG_URL),$(LIB_ROOT)/$(FFMPEG_TB),$(FFMPEG_SHA1))
 	$(call UNTAR,$(FFMPEG_TB),j)
 	$(call PATCH,$(FFMPEG),$(FFMPEG_DIFFS))
 	@touch $@
diff -Naur a/Makefile.hdhomerun b/Makefile.hdhomerun
--- a/Makefile.hdhomerun	2021-08-08 13:33:12.920651143 +0300
+++ b/Makefile.hdhomerun	2022-01-14 17:37:12.689123071 +0200
@@ -77,7 +77,7 @@
 	$(AR) rcs $@ $^
 
 $(LIB_ROOT)/$(LIBHDHR)/.tvh_download:
-	$(call DOWNLOAD,$(LIBHDHR_URL),$(LIB_ROOT)/$(LIBHDHR_TB),$(LIBHDHR_SHA1))
+#	$(call DOWNLOAD,$(LIBHDHR_URL),$(LIB_ROOT)/$(LIBHDHR_TB),$(LIBHDHR_SHA1))
 	$(call UNTAR,$(LIBHDHR_TB),z)
 	ln -sf libhdhomerun $(LIB_ROOT)/$(LIBHDHR)
 	$(call PATCH,$(LIBHDHR),$(LIB_DIFFS))
