diff -Naur a/bootstrap.mk b/bootstrap.mk
--- a/bootstrap.mk	2016-12-03 07:44:23.015041554 +0200
+++ b/bootstrap.mk	2018-05-03 23:16:16.754143315 +0300
@@ -1,6 +1,6 @@
 BOOTSTRAP_SUBDIRS += configure.ac
 BOOTSTRAP_SUBDIRS += lib/cpluff/configure.ac
-BOOTSTRAP_SUBDIRS += lib/gtest/configure.ac
+#BOOTSTRAP_SUBDIRS += lib/gtest/configure.ac
 
 BOOTSTRAP_TARGETS=$(basename $(BOOTSTRAP_SUBDIRS))
 all: $(BOOTSTRAP_TARGETS)
diff -Naur a/configure.ac b/configure.ac
--- a/configure.ac	2017-10-25 20:35:28.632974382 +0300
+++ b/configure.ac	2018-05-03 23:16:16.758143325 +0300
@@ -2391,18 +2391,11 @@
 ], [0])
 
 XB_CONFIG_MODULE([lib/gtest], [
-  ./configure \
     CC="$CC" \
     CXX="$CXX" \
     CFLAGS="$CFLAGS" \
     CXXFLAGS="$CXXFLAGS" \
-    --prefix="${prefix}" --includedir="${includedir}" --libdir="${libdir}" --datadir="${datadir}" \
-    --host=$host_alias \
-    --build=$build_alias \
-    --target=$target_alias \
-    --disable-shared \
-    --enable-static \
-    --with-pthreads
+    cmake .
 ], [$SKIP_CONFIG_GTEST])
 
 AC_OUTPUT
diff -Naur a/lib/libdvd/Makefile.in b/lib/libdvd/Makefile.in
--- a/lib/libdvd/Makefile.in	2016-12-03 07:44:23.515045133 +0200
+++ b/lib/libdvd/Makefile.in	2018-05-03 23:16:16.758143325 +0300
@@ -12,7 +12,7 @@
 else
   DVDNAV_A = @abs_top_srcdir@/tools/depends/target/libdvdnav/dvdnav-install/lib/libdvdnav.a
   DVDREAD_A = @abs_top_srcdir@/tools/depends/target/libdvdread/dvdread-install/lib/libdvdread.a
-  DVDCSS_A = @abs_top_srcdir@/tools/depends/target/libdvdcss/dvdcss-install/lib/libdvdcss.a
+  DVDCSS_A =
 endif
 
 SYSDIR = @abs_top_srcdir@/system/players/VideoPlayer
@@ -36,7 +36,7 @@
 ifeq ($(ARCH),arm-osx)
   BUILD_DVDCSS = 0
 else
-  BUILD_DVDCSS = 1
+  BUILD_DVDCSS = 0
 endif
 ifeq ($(BUILD_DVDCSS),1)
   SOS += libdvdcss-$(ARCH).so
diff -Naur a/tools/depends/native/TexturePacker/Makefile b/tools/depends/native/TexturePacker/Makefile
--- a/tools/depends/native/TexturePacker/Makefile	2016-12-03 07:44:23.699046450 +0200
+++ b/tools/depends/native/TexturePacker/Makefile	2018-05-03 23:16:16.758143325 +0300
@@ -9,13 +9,13 @@
 
 ifeq ($(NATIVEPLATFORM),)
   PLATFORM = native
-  EXTRA_CONFIGURE = --enable-static
+#  EXTRA_CONFIGURE = --enable-static
 else
   PLATFORM = $(NATIVEPLATFORM)
 endif
 
 ifeq ($(NATIVE_OS), linux)
-  EXTRA_CONFIGURE = --enable-static
+#  EXTRA_CONFIGURE = --enable-static
 endif
 ifeq ($(NATIVE_OS), android)
   EXTRA_CONFIGURE = --enable-static
diff -Naur a/xbmc/cores/VideoPlayer/DVDInputStreams/DllDvdNav.h b/xbmc/cores/VideoPlayer/DVDInputStreams/DllDvdNav.h
--- a/xbmc/cores/VideoPlayer/DVDInputStreams/DllDvdNav.h	2016-12-03 07:44:23.783047052 +0200
+++ b/xbmc/cores/VideoPlayer/DVDInputStreams/DllDvdNav.h	2018-05-03 23:16:16.758143325 +0300
@@ -34,6 +34,8 @@
  #define HAVE_CONFIG_H
  #endif
 
+ // pull in Kodi's config h instead of dvdnav's one
+ #include "xbmc/config.h"
  #include "dvdnav/dvdnav_internal.h"
  #include "dvdnav/vm.h"
  #include "dvdnav/dvd_types.h"
