diff -Naur a/xbmc/cores/DllLoader/DllLoader.h b/xbmc/cores/DllLoader/DllLoader.h
--- a/xbmc/cores/DllLoader/DllLoader.h	2017-05-25 23:27:13.202732089 +0300
+++ b/xbmc/cores/DllLoader/DllLoader.h	2018-05-03 23:19:03.366647563 +0300
@@ -28,6 +28,7 @@
     !defined(__arm__) && \
     !defined(__aarch64__) && \
     !defined(__mips__) && \
+    !defined(__s390x__) && \
     !defined(__SH4__) && \
     !defined(__sparc__) && \
     !defined(__arc__) && \
diff -Naur a/xbmc/cores/DllLoader/ldt_keeper.c b/xbmc/cores/DllLoader/ldt_keeper.c
--- a/xbmc/cores/DllLoader/ldt_keeper.c	2017-05-25 23:27:13.202732089 +0300
+++ b/xbmc/cores/DllLoader/ldt_keeper.c	2018-05-03 23:19:03.366647563 +0300
@@ -24,6 +24,7 @@
     !defined(__arm__) && \
     !defined(__aarch64__) && \
     !defined(__mips__) && \
+    !defined(__s390x__) && \
     !defined(__SH4__) && \
     !defined(__sparc__) && \
     !defined(__arc__) && \
diff -Naur a/xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.h b/xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.h
--- a/xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.h	2017-05-25 23:27:13.202732089 +0300
+++ b/xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.h	2018-05-03 23:19:03.370647578 +0300
@@ -299,6 +299,7 @@
     !defined(__arm__) && \
     !defined(__aarch64__) && \
     !defined(__mips__) && \
+    !defined(__s390x__) && \
     !defined(__SH4__) && \
     !defined(__sparc__) && \
     !defined(__arc__) && \
diff -Naur a/xbmc/threads/Atomics.cpp b/xbmc/threads/Atomics.cpp
--- a/xbmc/threads/Atomics.cpp	2017-05-25 23:27:13.202732089 +0300
+++ b/xbmc/threads/Atomics.cpp	2018-05-03 23:19:03.370647578 +0300
@@ -110,6 +110,7 @@
     defined(__powerpc__) || \
     defined(__arm__) || \
     defined(__aarch64__) || \
+    defined(__s390x__) || \
     defined(__SH4__) || \
     defined(__sparc__) || \
     defined(__arc__) || \
diff -Naur a/xbmc/threads/Atomics.h b/xbmc/threads/Atomics.h
--- a/xbmc/threads/Atomics.h	2016-12-03 07:44:23.979048454 +0200
+++ b/xbmc/threads/Atomics.h	2018-05-03 23:19:03.370647578 +0300
@@ -22,7 +22,7 @@
 
 //! @todo Inline these methods
 long cas(volatile long *pAddr, long expectedVal, long swapVal);
-#if !defined(__ppc__) && !defined(__powerpc__) && !defined(__arm__)
+#if !defined(__ppc__) && !defined(__powerpc__) && !defined(__arm__) && !defined(__s390x__)
 long long cas2(volatile long long* pAddr, long long expectedVal, long long swapVal);
 #endif
 long AtomicIncrement(volatile long* pAddr);
diff -Naur a/xbmc/utils/MathUtils.h b/xbmc/utils/MathUtils.h
--- a/xbmc/utils/MathUtils.h	2017-05-25 23:27:13.202732089 +0300
+++ b/xbmc/utils/MathUtils.h	2018-05-03 23:19:03.366647563 +0300
@@ -35,6 +35,7 @@
 #if defined(__ppc__) || \
     defined(__powerpc__) || \
     defined(__mips__) || \
+    defined(__s390x__) || \
     defined(__arm__) || \
     defined(__aarch64__) || \
     defined(__SH4__) || \
