diff -Naur a/configure.ac b/configure.ac
--- a/configure.ac	2017-10-25 20:35:28.632974382 +0300
+++ b/configure.ac	2018-05-03 23:20:27.606963401 +0300
@@ -2203,7 +2203,6 @@
     tools/Linux/kodi-standalone.sh \
     tools/Linux/kodi-xsession.desktop \
     tools/EventClients/Makefile \
-    tools/EventClients/Clients/OSXRemote/Makefile \
     xbmc/peripherals/addons/Makefile \
     xbmc/peripherals/bus/Makefile \
     xbmc/peripherals/devices/Makefile \
