diff -Naur a/xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp b/xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp
--- a/xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp	2017-02-25 10:50:51.056577856 +0200
+++ b/xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp	2018-05-03 23:31:35.830301063 +0300
@@ -486,7 +486,7 @@
         // guess next pts
         m_audioClock += audioframe.duration;
 
-        int ret = m_pAudioCodec->Decode(nullptr, 0, DVD_NOPTS_VALUE, DVD_NOPTS_VALUE);
+        int ret = 0;
         if (ret < 0)
         {
           CLog::Log(LOGERROR, "CVideoPlayerAudio::DecodeFrame - Decode Error. Skipping audio packet (%d)", ret);
