diff -Naur a/xbmc/windowing/X11/GLContextEGL.cpp b/xbmc/windowing/X11/GLContextEGL.cpp
--- a/xbmc/windowing/X11/GLContextEGL.cpp	2016-12-03 07:44:24.019048740 +0200
+++ b/xbmc/windowing/X11/GLContextEGL.cpp	2018-05-03 23:35:21.063605973 +0300
@@ -409,9 +409,11 @@
     m_eglConfig=eglConfig;
 
     XVisualInfo x11_visual_info_template;
-    if (!eglGetConfigAttrib(m_eglDisplay, m_eglConfig, EGL_NATIVE_VISUAL_ID, (EGLint*)&x11_visual_info_template.visualid)) {
+    EGLint visualID;
+    if (!eglGetConfigAttrib(m_eglDisplay, m_eglConfig, EGL_NATIVE_VISUAL_ID, &visualID)) {
       CLog::Log(LOGERROR, "Failed to query native visual id\n");
     }
+    x11_visual_info_template.visualid = visualID;
     int num_visuals;
     return XGetVisualInfo(m_dpy,
                         VisualIDMask,
